flask-server/URLgenerator.py

```diff
@@ -0,0 +1,28 @@
+
+
+import string
+import random
+
+# alphabetLower = list(string.ascii_lowercase)
+# alphabetUpper = list(string.ascii_uppercase)
+# print(alphabetLower)
+# print(alphabetUpper)
+
+alphabet = list(string.ascii_letters)
+
+def get_random_string(length):
```

Comment:
You should practice annotating Python code. For example:
```
def function_name(long_url: str)
```
This is the same in Java:
```
public void my_function(string long_url){
   // do stuff here
}
```